src/ctest.ts

```diff
@@ -346,22 +346,28 @@ export class CTestDriver implements vscode.Disposable {
     this._decorationManager.clearFailingTestDecorations();
 
     const configuration = driver.currentBuildType;
-    const child
-        = driver.executeCommand(await paths.ctestPath,
-                                [`-j${config.numCTestJobs}`, '-C', configuration, '-T', 'test', '--output-on-failure']
-                                    .concat(config.ctestArgs),
-                                new CTestOutputLogger(),
-                                {environment: config.testEnvironment, cwd: driver.binaryDir});
-
-    const res = await child.result;
-    await this.reloadTests(driver);
-    if (res.retc === null) {
-      log.info('CTest run was terminated');
-      return -1;
+    const ctestpath = await paths.ctestPath;
+    if (ctestpath) {
```

Comment:
Swap the if-else here. Check if `ctest_path` is null and return early, then the larger code branch can be unindented and there is not need for an `else` branch.